frontend/projects/upgrade/src/app/features/dashboard/experiments/modals/upsert-experiment-modal/upsert-experiment-modal.component.ts

```diff
@@ -340,8 +340,14 @@ export class UpsertExperimentModalComponent implements OnInit, OnDestroy {
 
   listenForPrimaryButtonDisabled() {
     this.isPrimaryButtonDisabled$ = this.isLoadingUpsertExperiment$.pipe(
-      combineLatestWith(this.isInitialFormValueChanged$),
-      map(([isLoading, isInitialFormValueChanged]) => isLoading || !isInitialFormValueChanged)
+      combineLatestWith(
+        this.isInitialFormValueChanged$,
+        this.experimentForm.statusChanges.pipe(startWith(this.experimentForm.status))
+      ),
```

Comment:
i've actually made a similar change in my PR to put the ts_config in there, thing is we don't need to combine and query the statusChanges observable, simply `!experimentForm.valid` will give the same thing